backend/tests/services/test_skill_marketplace_service.py

```
@@ -0,0 +1,543 @@
+            test_db,
+            marketplace_skill_id=marketplace_skill_id,
+            user_id=test_admin_user.id,
+        )
+
+        # Get collections
+        collections = collection_service.get_my_collections(
+            test_db, user_id=test_admin_user.id
+        )
+
+        assert len(collections) == 1
+        assert collections[0].is_available == True
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `collections[0].is_available:` for truth checks

Replace with `collections[0].is_available`

(E712)